### DISCUSSION TITLE
Class COM not found

### DISCUSSION THREAD

**Discussion by loomy2021** [livewire/livewire#5594] [1. Help]
HI all,

I've activated the 
[COM_DOT_NET]
 extension=php_com_dotnet.dll
 enable_dl = On in my php.ini and it works fine if I run the code 
      @php
         $word = new com("word.application") or die("Konnte Word nicht instanziieren");
      @endphp
in a simple blade file like welcome.blade in my Laravel project.
But if I try to run this code in a method from my Livewire component I get the error: Class COM not found. What can I do to get this working in a livewire component, does anyone know what I am missing? 
I need to work with some external DLL Files on Windows env.- like the word.application

Thank you!

**Comment by joshhanley** [comment]
@loomy2021 you will need to import the `com()` function in your component class for it to be able to be used in a component method.

Hope this helps!
